packages/app/src/components/MenuBar/MenuBar.js

```diff
@@ -35,11 +36,11 @@ export const MenuBar = (props, context) => (
             onTranslate={() => console.log('translate callback')}
             onError={() => console.log('error!')}
         />
-        <VisualizationOptionsManager />
-        <div>Download</div>
-        <div>Embed</div>
+        <VisualizationOptionsManager labelStyle={styles.label} />
+        <div style={styles.label}>Download</div>
+        <div style={styles.label}>Embed</div>
```

Comment:
Shall we just take out the "Embed" option since we are not implementing it in 2.31?